draft-ietf-tls-tls13.md

```diff
@@ -5110,7 +5132,7 @@ ClientHello, otherwise falling back to a 1-RTT handshake.
 This is necessary for the ClientHello storage mechanism
 described in {{client-hello-recording}} because otherwise the server
 needs to store an unlimited number of ClientHellos and is a useful optimization for
-single-use tickets because it allows efficient rejection of ClientHellos
+self-contained single-use tickets because it allows efficient rejection of ClientHellos
```

Comment:
tickets that are lookup keys to a database will need to be looked up before they could be used, and if you can't find it, you know it's not usable at all, let alone for 0-RTT.